manage-data/data-store/data-streams/failure-store.md

```diff
@@ -90,6 +90,12 @@ PUT _data_stream/my-datastream-existing/_options
 
 1. Redirecting failed documents into the failure store will now be disabled.
 
+:::{tip}
+:applies_to: {"stack": "ga 9.2, preview 9.1", "serverless": "ga"}
+
+You can also enable the data stream failure store in {{kib}}. Locate the data stream on the **Streams** page, where a stream maps directly to a data stream. Select a stream to view its details and go to the **Retention** tab where you can find the **Enable failure store** option.
+:::
```

Comment:
Here we have `a stream maps directly...` and on the [Streams](https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/3399/solutions/observability/streams/streams) page in the Obs docs we have `A Stream directly corresponds to...`. 

I'm not sure which is better but it'd be good to make sure we're consistent throughout the docs on "Stream" versus "stream".

Reply:
Just saw your other comment and suggestion to use Streams page instead, so will do that